packages/Search/src/details/DTK_DetailsAlgorithms.hpp

```diff
@@ -64,7 +64,7 @@ bool isValid( Box const &b )
 KOKKOS_INLINE_FUNCTION
 bool isValid( Sphere const &s )
 {
-    using KokkosHelpers::isFinite;
+    using KokkosExt::isFinite;
```

Comment:
Why have the `using` here? To clean up the following line?

Reply:
Yes that was the intention.